docs/source/backends/llamacpp.md

```diff
@@ -75,13 +64,31 @@ docker run \
     --gpus all \
     -p 3000:3000 \
     -e "HF_TOKEN=$HF_TOKEN" \
-    -v "$HOME/models:/models" \
+    -v "$HOME/models:/app/models" \
     tgi-llamacpp \
     --n-gpu-layers 99
+    --model-id "Qwen/Qwen2.5-3B-Instruct"
+```
+
+## Using a custom GGUF
+
+GGUF files are optional as they will be automatically generated at
+startup if not already present in the `models` directory. However, if
+the default GGUF generation is not suitable for your use case, you can
+provide your own GGUF file with `--model-gguf`, for example:
```

Comment:
Is that a local file reference that users need to mount themselves ?
How does this work with file splitting : https://huggingface.co/unsloth/DeepSeek-R1-GGUF/tree/main ?
It's saying we need both, why ? 

Seems like we could potentially keep feeding a Hub link no ? `unsloth/DeepSeek-R1-GGUF/resolve/main/DeepSeek-R1-Q2_K/` (File splitting makes this a mess, and potentially various models don't have the same layout, I'm just thinking how to deploy easily).

Reply:
Also in the unspecfied gguf file case, are we saving the model in the mounted volume (so that subsequent startups are faster ? )

Reply:
yes, if you omit the `--model-gguf` option, the generated gguf will be stored in `/app/models` and will be used the next time.